common/app/views/support/EmailHelpers.scala

```diff
@@ -53,12 +53,16 @@ object EmailHelpers {
     s"""<img src="${Static(s"images/email/icons/$name.png")}" class="icon icon-$name">"""
   }
 
-  def img(src: String, alt: Option[String] = None) = Html {
-    s"""<img width="580" class="full-width" src="$src" ${alt.map(alt => s"""alt="$alt"""").getOrElse("")}>"""
+  def imgForArticle(src: String, alt: Option[String] = None) = Html {
+    s"""<img width="${EmailImage.knownWidth}" class="full-width" src="$src" ${alt.map(alt => s"""alt="$alt"""").getOrElse("")}>"""
+  }
+
+  def imgForFront(src: String, alt: Option[String] = None) = Html {
+    s"""<img width="${FrontEmailImage.knownWidth}" class="full-width" src="$src" ${alt.map(alt => s"""alt="$alt"""").getOrElse("")}>"""
```

Comment:
My example had the params the wrong way round, but `img(width)` was intended to return `(String, Option[String]) => Int`.

    def img(knownWidth: Int)(src: String, alt: Option[String] = None) = Html { ... }
    def imgForFront = img(FrontEmailImage.knownWidth)
    def imgForArticle = img(EmailImage.knownWidth)